src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " start working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(getName() + " stop working");
+    }
```

Comment:
```suggestion
    @Override
    public void doWork() {
        System.out.println(getClass().getSimpleName() + " start working");
    }

    @Override
    public void stopWork() {
        System.out.println(getClass().getSimpleName() + " stop working");
    }
```

Если хочешь подтягивать имя класса, можно использовать мутод getClass().getSimpleName(), тогда не нужно создавать дополнительное поле в классе Machine